sdk/src/test/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/connector/ApiConnectorTest.java

```diff
@@ -313,6 +318,21 @@ public void testGetAnnouncementData_ApiReturnsAuthenticationError() throws Excep
         classUnderTest.getAnnouncementData(apiNotification);
     }
 
+    @Test
+    public void testGetOrderStatus() throws Exception {
+        String orderId = "3dfe639-967a-473d-8642-fa9347223d7a";
+        JSONObject orderStatus = new JSONObject().put("order", getOrderStatusResult());
+
+        when(jsonTemplate.get("v2/orders/" + orderId, "token")).thenReturn(orderStatus);
+
+        OrderStatusResponse actualResponse = classUnderTest.getOrderStatus(orderId, "token");
+        OrderStatusResult actualResult = actualResponse.getOrderStatusResult();
+
+        assertThat(actualResult.getMerchantOrderId(), is("25da863a-60a5-475d-ae47-c0e4bd1bec31"));
+        assertThat(actualResult.getId(), is("ORDER1"));
+        assertThat(actualResult.getOrderStatus(), is("COMPLETED"));
```

Comment:
Can you add assertions for all the fields here?

Reply:
Done.